### PR TITLE
Support multiple custom homebrew taps

### PR DESCRIPTION
**Pull request by philmcmahon**
Just raising this as a point of discussion - it might go against the out of scope feature of `Installing Homebrew formulae by default for everyone in an organisation`.

At my organisation we are using strap to install quite a large custom formula to set up developer machines with software they'll need to do their jobs. As part of this we want to install [adoptopenjdk8](https://github.com/AdoptOpenJDK/homebrew-openjdk). You can see the cask here https://github.com/guardian/homebrew-devtools/blob/master/Casks/gu-base.rb

Adoptopenjdk now relies on a custom tap for versions other than the latest. We're already using CUSTOM_HOMEBREW_TAP to add our custom tap `guardian/homebrew-devtools` to get the above cask, but now need to also install a second tap.

We have a fork of strap we could add this functionality too but I thought I'd raise it here to see if you though it was a worthwhile feature.

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
Thanks for the PR @philmcmahon!

The recommended approach for "we want to bulk-install some applications for people" is generally "have them listed in the `Brewfile`s for the relevant projects" but I appreciate that y'all clearly disagree (although I'd love to hear why so I can consider these use-cases in future).

This approach would also allow you to specify as many custom taps as you want/need. Perhaps it would be worth considering something like a `CUSTOM_BREWFILE` or something that could be used to bulk-install in this way?

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Strap is also available as a Docker image on [Docker Hub (`mikemcquaid/strap`)](
 - `WEB_CONCURRENCY`: the number of Unicorn (web server) processes to run (defaults to 3).
 - `STRAP_ISSUES_URL`: the URL where users should file issues (defaults to no URL).
 - `STRAP_BEFORE_INSTALL`: instructions displayed in the web application for users to follow before installing Strap (wrapped in `<li>` tags).
-- `CUSTOM_HOMEBREW_TAP`: an optional Homebrew tap to install with `brew tap`. Specify multiple arguments to brew tap by separating values with spaces.
+- `CUSTOM_HOMEBREW_TAPS`: optional Homebrew taps to install with `brew tap`. Specify multiple arguments to brew tap by separating values with spaces. Specify multiple taps by separating values with commas.
```
[the remark this answers is not in the document]
Comment (philmcmahon) [reply README.md:71]: Good shout, I'll fix this

**Comment by philmcmahon** [comment]
Thanks for the reply @MikeMcQuaid ! I made this PR right before going off on holiday for a week. I will have a think about whether we can use Brewfiles instead and get back to this next week.

**Comment by MikeMcQuaid** [comment]
Sounds good! I'm on holiday next week too just to set expectations 😁.

Definitely want to work with y'all to figure out how to upstream as much as possible that you're doing. It's good to have a prominent non-github customer for Strap that's open to working with upstream (me!) to make it better.

**Comment by philmcmahon** [comment]
Hey Mike, sorry for the radio silence here! I think I'm going to close this for now, since having Brewfiles in repos is something we've started to pick up at the guardian. 

I still think there's a use case for having a 'bulk install' when someone joins an organisation - which is what we use strap for. Getting people to run `brew bundle` is fairly painless but still pretty slow - when most of our projects are scala based just having java and scala preinstalled usually means you can get straight in and start compiling the code without any installation steps. Having said that, we're then relying on someone having their laptop in the special 'guardian way' which isn't very open source friendly, so one Brewfile per repo seems to make more sense. 

It would be great if we can help out with strap a bit as we're a major user still. I think the most likely time we'll have a resurgence of interest in laptop bootstrapping is probably 2022 when we next renew our machines!

**Pull request closed without merging**